### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Astijq**
cia mano take on our app description

### CODE REVIEW

**Review by Astijq — COMMENTED** [review]
o cia?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Trash-bolt
 write description
+Astijus description
```
[the remark this answers is not in the document]
Comment (kambalas) [reply README.md:3]: ok

**Review by kambalas — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (neringamaj) [reply README.md:3]: puiku

**Review by neringamaj — APPROVED** [review]
good work

**Review by neringamaj — APPROVED** [review]

**Pull request merged**